pkg/controllermanager/controller/controllerregistration/seed_control_test.go

```diff
@@ -146,7 +126,6 @@ var _ = Describe("controllerRegistrationReconciler", func() {
 		}
 		backupEntryList = &gardencorev1beta1.BackupEntryList{
 			Items: []gardencorev1beta1.BackupEntry{
-				*backupEntry1,
 				*backupEntry2,
 				*backupEntry3,
```

Comment:
Nit: as `backupEntry1 ` is now removed we could rename the other vars to keep the order. 
```suggestion
				*backupEntry1,
				*backupEntry2,
```